lib/gollum/public/gollum/stylesheets/gollum.scss

```diff
@@ -46,14 +47,24 @@ a {
     float: left;
     line-height: normal;
     margin: 0;
-    margin-top: 2em;
+    margin-top: 3em;
```

Comment:
This affects the margins for all h1's, right? Have you tested how that looks, and would it be an alternative to add the extra spacing only to the first h1?

Reply:
> This affects the margins for all h1's, right?

No, only the h1 within the head div.